web/resources/product/admin/views/AdminCreateProduct.js.jsx

```diff
@@ -55,10 +55,10 @@ class AdminCreateProduct extends Binder {
     /**
      * This let's us change arbitrarily nested objects with one pass
      */
-    let newState = _.update(this.state, e.target.name, function() {
+    const newState = _.update(_.cloneDeep(this.state), e.target.name, () => {
```

Comment:
why `const` instead of `let`?

Reply:
Because we never reassign it, it doesn't need to be `let`. Although it works either way.